flow/util/makeIssue.sh

```diff
@@ -66,6 +66,7 @@ ISSUE_CP_FILES+="${ISSUE_CP_FILES_PLATFORM} \
 
 echo "Creating ${RUN_ME_SCRIPT} script"
 echo "#!/usr/bin/env bash"                     >  ${RUN_ME_SCRIPT}
+echo "export FLOW_HOME=\${FLOW_HOME:-\$(pwd)}" >> ${RUN_ME_SCRIPT}
```

Comment:
I think that the variable should be added to the exclude list to avoid setting it twice.

Reply:
It isn't set in the case where I'm building a projects outside of the ORFS flow folder. See below.

Reply:
That's ok. My concern is where the variable is set and would lead to different behavior.